test/weak_convergence/iip_weak.jl

```diff
@@ -40,9 +52,9 @@ sim = test_convergence(dts,prob,SROCK1(),save_everystep=false,trajectories=Int(4
 #@test abs(sim.𝒪est[:weak_l2]-1) < 0.3
 #@test abs(sim.𝒪est[:weak_l∞]-1) < 0.3
 println("SROCK2")
-sim = test_convergence(dts,prob,SROCK2(),save_everystep=false,trajectories=Int(4e4),
+sim = test_convergence(dts,prob,SROCK2(),save_everystep=false,trajectories=Int(1e5),
                         weak_timeseries_errors=false)
-@test abs(sim.𝒪est[:weak_final]-1.5) < 0.3
+@test abs(sim.𝒪est[:weak_final]-1.5) < 0.31
```

Comment:
seems like a real test error here? Odd...

Reply:
Feel free to move tests into the BuildKite queue to speed them up. We can make them be highly multithreaded and such if we need to.

Reply:
Do the weak adaptive tests already run on the fastest GPUs/CPUs (buildkite/stochasticdiffeq-dot-jl/weakconvergen..)? Looks like they got aborted because of their runtime.

Reply:
Yes they were on one of the fast machines with a GPU. Were they extended?

Reply:
Not yet..

Should I just try increasing
https://github.com/SciML/StochasticDiffEq.jl/blob/a96819fca0fccf29f4210e679bb36b6e7c50ef64/.buildkite/pipeline.yml#L32
if they fail again or is it better to split the file/try to make the tests faster?

Reply:
yeah double it and see what happens.